### PR TITLE
Prompt descriptions #2703

### PR DESCRIPTION
**Pull request by mattkrick**
TEST
- [x] existing teams have descriptions for the stock questions
- [x] new teams have descriptions for the stock questions

MODAL
- [ ] a team member can add a description to a prompt via the template manager modal
- [ ] hitting the add new prompt button creates a new prompt & autofocuses on the prompt question
- [ ] hitting tab goes down from the prompt question to the prompt description
- [ ] Adding a new template works

### CODE REVIEW

**Comment by jordanh** [comment]
# Test Cycle 1

## Tests
- [ ] existing teams have descriptions for the stock questions
- [ ] new teams have descriptions for the stock questions
- [ ] a team member can add a description to a prompt via the template manager modal

File: src/server/database/migrations/20190327165913-promptDescriptions.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    'Wind',
+    'Anchors',
+    'Rocks',
+    'Start',
+    'Stop',
+    'Continue',
+    'What’s working?',
+    'Where did you get stuck?'
+  ]
+  const descriptions = [
```
Comment (jordanh) [inline src/server/database/migrations/20190327165913-promptDescriptions.js]: Great job on these descriptions!

**Comment by mattkrick** [comment]
@jordanh you may need to rimraf your node modules & lockfile to get rid of those typescript errors. for example, if you open MediaSwarm.ts, you'll see `addStreams` does exist, so something is funky

**Comment by ackernaut** [comment]
Here are [updated designs](https://www.figma.com/file/d2OGg3CGIPCBb5IIGf40x6to?node-id=134%3A2428&utm_campaign=team&utm_source=slack&utm_medium=message&utm_content=file_comment#5635867) for the template editor with fields for the descriptions

**Comment by mattkrick** [comment]
@jordanh ready for review

**Comment by jordanh** [comment]
# Test Cycle 1

## Tests
### Migrations
- [ ] can migrate down & up
- [ ] existing teams have descriptions for the stock questions
- [ ] new teams have descriptions for the stock questions

### Modal
- [ ] a team member can add a description to a prompt via the template manager modal
- [ ] hitting the add new prompt button creates a new prompt & autofocuses on the prompt question
- [ ] hitting tab goes down from the prompt question to the prompt description
- [ ] Adding a new template works

## Notes

**Comment by jordanh** [comment]
@mattkrick ran above, added a couple of test cases. Saw the demo didn't have the updated prompts.

**Comment by mattkrick** [comment]
demo fixed, merging

**Pull request merged**